include/alpaka/core/BoostPredef.hpp

```diff
@@ -19,8 +19,7 @@
 #        include <hip/hip_runtime.h>
 // HIP defines "abort()" as "{asm("trap;");}", which breaks some kernels
 #        undef abort
-// there is no HIP_VERSION macro
-#        define BOOST_LANG_HIP BOOST_VERSION_NUMBER_AVAILABLE
+#        define BOOST_LANG_HIP BOOST_VERSION_NUMBER(HIP_VERSION_MAJOR, HIP_VERSION_MINOR, 0)
```

Comment:
Could you please add `HIP_VERSION_PATCH` instead of `0` for the last number.

Reply:
I thought about, but `HIP_VERSION_PATCH` is too large: `BOOST_VERSION_NUMBER` leaves four digits for the patch number, but _e.g._ in ROCm 5.0.x `HIP_VERSION_PATCH` is `13602` :-/

Reply:
And it's the same in 5.0.0, 5.0.1 and 5.0.2, so it would not be useful to distinguish them, either.

Reply:
ok makes sense to keep it in this case as `0`